examples/prompty/basic/prompty-quickstart.ipynb

```diff
@@ -79,10 +79,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from promptflow.core import Flow\n",
+    "from promptflow.core import Prompty\n",
```

Comment:
please change sample in separte PR: https://github.com/microsoft/promptflow/pull/2856